demos/pet-demo/src/components/__isograph/AdItem/AdItemDisplay/refetch_reader.ts

```diff
@@ -1,17 +1,17 @@
-import type { RefetchReaderArtifact, ReaderAst, RefetchQueryNormalizationArtifact } from '@isograph/react';
+import type { ReaderAst, RefetchQueryNormalizationArtifact, RefetchReaderArtifact } from '@isograph/react';
 const includeReadOutData = (variables: any, readOutData: any) => {
   variables.id = readOutData.id;
   return variables;
 };
 
-import { makeNetworkRequest, wrapResolvedValue, type IsographEnvironment, type FragmentReference, type RefetchQueryNormalizationArtifactWrapper, type DataId, type TopLevelReaderArtifact } from '@isograph/react';
+import { makeNetworkRequest, wrapResolvedValue, type FragmentReference, type IsographEnvironment, type Link, type RefetchQueryNormalizationArtifactWrapper, type TopLevelReaderArtifact } from '@isograph/react';
```

Comment:
why is this changing? this comes from a call to `format!`, but we don't re-order `type IsographEnvironment` and `type FragmentReference` anywhere??

Reply:
This got changed by my editor, it's fixed